src/_guides/language/language-tour.md

```diff
@@ -3564,6 +3564,8 @@ defines the unimplemented method (abstract is OK),
 and the dynamic type of the receiver has an implementation of `noSuchMethod()`
 that's different from the one in class `Object`.
 
+Additonally, you **can't invoke** private members of an implemented interface, **ever**.
```

Comment:
I don't think this is accurate.  You *can* invoke private members of an implemented interface, if you are in the same library (and hence you can mention the private name), e.g. this totally works:

```dart
class C {
  void _privateMember() {
    print('C._privateMember() called');
  }
}

class D implements C {
  @override
  dynamic noSuchMethod(Invocation invocation) {
    print('D._noSuchMethod() called');
  }
}

main() {
  C()._privateMember(); // Prints `C._privateMember() called`
  D()._privateMember(); // Prints `D._noSuchMethod() called`
}
```

The new behaviour from https://github.com/dart-lang/sdk/issues/49687 doesn't change whether you can invoke a private member of an implemented interface; it just changes what happens if you do, under one very specific circumstance: if you invoke a private member of a class, and that class has an implemention of `noSuchMethod` but doesn't implement that member, _and that member is private to a different library than the one the class is declared in_, then instead of the invocation being routed to your `noSuchMethod` member, an exception will be thrown.

That's quite a mouthful, and honestly it's quite a rare situation.  IMHO there's no need to change the language tour to mention this minor little detail, because the number of people are going to run across it is really quite low.

Reply:
Ok thanks for explaining! I figured it was unnecessary but just wanted to make sure. Closing this